ultravox/inference/base.py

```diff
@@ -40,6 +40,15 @@ def infer(
     ) -> VoiceOutput:
         pass
 
+    def batch_infer(
+        self,
+        samples: List[datasets.VoiceSample],
+        batch_size: int = 1,
+        max_tokens: Optional[int] = None,
+        temperature: Optional[float] = None,
+    ):
+        return self.batch_infer(samples, batch_size, max_tokens, temperature)
```

Comment:
I'm confused, what does this function do? Did you want it to be abstract and/or to call `infer` repeatedly instead?

Reply:
Yeah, I'm just going to add a type check in infer_tool.dataset_infer, and i can get rid of this function.